Dockerfile

```diff
@@ -98,6 +98,7 @@ COPY config/runtime.exs config/
 
 # Generate swagger V2 file
 RUN mix run --no-start -e 'IO.puts(Mix.Project.config[:version])' >AEMDW_VERSION
+ARG PATH_PREFIX
```

Comment:
I don't think ARGs in dockerfiles are env variables. This wouldn't be accessed on the swagger script

Reply:
They are, for `RUN` commands: https://docs.docker.com/reference/dockerfile/#using-arg-variables . If you wanna test it, you can rebuild the docker container using the `docker-compose-dev.yml` included in the PR

Reply:
You're right, it works the same! sorry, merging